lib/types/types.dart

```diff
@@ -50,13 +50,16 @@ class Document extends Node<BLOCKS> {
 
 class TextNode extends Node<String> {
   String _nodeType = 'text';
-  String value;
-  List<Mark> marks;
-
-  TextNode(dynamic node)
-      : value = node['value'],
-        _nodeType = node['nodeType'],
-        marks = node['marks']?.map((mark) => Mark(mark['type'])).toList() ?? [];
+  String value = '';
+  List<Mark> marks = [];
+
+  TextNode(dynamic node) { 
+    value = node['value'] ?? '';
+    _nodeType = node['nodeType'] ?? '';
+    node['marks']?.map((mark) {
+      marks.add(Mark(mark['type']));
+    });
+  }
```

Comment:
Any particular reason from changing this to a constructor body instead of an initializer list?

Reply:
I'm using the package in our app Fita https://play.google.com/store/apps/details?id=com.muna.lively (Flutter 2.2). During the parsing, It kept getting exception of casting between List< Mark> and List<dynamic>. 

The `node['marks']` will return a List<dynamic> while the `marks` is a List<Mark>, hence the parsing issue. What I did was to convert the List<dynamic> to List<Mark>.